### PR TITLE
Add switch case to router.hooks

### PR DESCRIPTION
**Pull request by JohanBester**
Hi Folks,

This PR is to add the SWITCH case statement to router.hooks

This PR needs to be merged to Development,
And then Development would need to be merged to master to get this change into the works before week 7

Kindly review and comment.
Thanks!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@ node_modules
 *.sw*
 .idea
 .env
-.parcel-cache
\ No newline at end of file
+.parcel-cache
+package-lock.json
```
Comment (matsinet) [inline .gitignore:9]: The package-lock.json needs to be committed so remove that line

File: index.js [hunk 2]
```diff
@@ -68,46 +65,57 @@ function afterRender(state) {
...
-          done();
-        });
-    } else {
-      done();
+    // Add a switch case statement to handle multiple routes
+    switch (view) {
+      case "Home": {
+        axios
+          .get(
+            `https://api.openweathermap.org/data/2.5/weather?appid=fbb30b5d6cf8e164ed522e5082b49064&q=st%20louis`
```
Comment (matsinet) [inline index.js:78]: Replace the openweathermap key `fbb30b5d6cf8e164ed522e5082b49064` with the .env variable `${process.env.OPEN_WEATHER_MAP_API_KEY}`

**Review by matsinet — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (JohanBester) [reply .gitignore]: Corrected this issue

File: index.js (hunk 2 above)
Comment (JohanBester) [reply index.js]: The API key has been replaced

**Review by matsinet — APPROVED** [review]

**Pull request merged**